readme.md

```diff
@@ -343,6 +343,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [Linux Containers](https://github.com/Friz-zy/awesome-linux-containers)
 - [Beginner-Friendly Projects](https://github.com/MunGell/awesome-for-beginners)
 - [Bluetooth Beacons](https://github.com/beaconinside/awesome-beacon)
+- [Spanish NLP](https://github.com/dav009/awesome-spanish-nlp)
```

Comment:
Acronyms are always common and well known for its target group, but it's not very approachable for outside people that might be interested. Never hurts to be explicit.